lib/src/components/NxDrawer/NxDrawer.scss

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @use '../../scss-shared/nx-container-helpers';
+ @use '../../scss-shared/nx-text-helpers';
+
+@keyframes slide-in {
+  // This is required to fix jittering in Firefox and Safari
+  // and not animating at all in Chrome.
```

Comment:
This comment is no longer applicable right?  It was originally on the `1%` block
```suggestion
```